internal/test/plugins/quota_generic.go

```diff
@@ -136,6 +136,20 @@ func (p *GenericQuotaPlugin) ScrapeRates(ctx context.Context, project core.Keyst
 	return result, string(serializedStateBytes), nil
 }
 
+func (p *GenericQuotaPlugin) BuildServiceUsageRequest(project core.KeystoneProject, allAZs []limes.AvailabilityZone) (*liquid.ServiceUsageRequest, error) {
+	return &liquid.ServiceUsageRequest{
+		AllAZs: []liquid.AvailabilityZone{"az-one", "az-two"},
+		ProjectMetadata: &liquid.ProjectMetadata{
+			UUID: "uuid-for-berlin",
+			Name: "berlin",
+			Domain: liquid.DomainMetadata{
+				UUID: "uuid-for-germany",
+				Name: "germany",
+			},
+		},
```

Comment:
This is kind of the opposite case. We don't need to make up any data if we get good data in the arguments:
```suggestion
		ProjectMetadata: project.ForLiquid(),
```